jumpcloud-ADMU/Powershell/Private/RegistryValidation/Backup-ProfileImageACL.ps1

```diff
@@ -0,0 +1,40 @@
+function Backup-ProfileImageACL {
+    [CmdletBinding()]
+    param (
+        [Parameter(Mandatory = $true, HelpMessage = "Full path to the user profile image folder.")]
+        [string]$ProfileImagePath,
+        [Parameter(Mandatory = $true, HelpMessage = "SID of the user.")]
+        [string]$sourceSID
+    )
+    begin {
+        $timestamp = Get-Date -Format "yyyyMMdd-HHmm"
+        $path = "$ProfileImagePath\AppData\Local\JumpCloudADMU"
+        $file = "$path\${sourceSID}_permission_backup_${timestamp}"
+        $ACLPermissionLogPath = "$(Get-WindowsDrive)\Windows\Temp\jcAdmu_ACL.log"
```

Comment:
I'll not want us to merge into master until we come up with a plan for these logs. We have a set of logs for setting ACLs and now this new one for backing up ACLs. 

When we merge 2.11.0 in, these should be one file imo. I have added the `jcadmu_ntfs` log file to temp but now thinking about it, having three different log files in that directory seems poorly planned out. 

I can create another card but I won't want to merge this to production until we consolidate the two `jcadmu_ntfs` & `jcadmu_ACL` logs. This can be an easy 1 pointer to just note the step being processed then save the log data into one log file @ `C:\Users\UserBeingMigrated\AppData\Local\JumpCloudADMU\aclLogs` (or similar) - team should discuss.